### ISSUE TITLE
RIP. Spotify has deprecated `audio-features`, `audio-analysis`, `recommendations` API's.

### ISSUE THREAD

**Issue by tusharkekw** [barkloaf/MetaGrab#4]

**Comment by barkloaf** [comment]
yeah, I noticed that. I'll probably get to fixing it when I have time, which will also probably mostly happen in the API repo.

**Comment by barkloaf** [comment]
looked into this more, and nevermind. this is the end of MetaGrab unfortunately. please see the deprecation notice in the readme. thank you for the support! I'm glad people enjoyed this project like I did
